src/DynamoCoreWpf/ViewModels/Core/GeometryScalingViewModel.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Collections.Generic;
+using Dynamo.Core;
+using Dynamo.ViewModels;
+
+namespace ViewModels.Core
+{
+    /// <summary>
+    /// This class will be contain information about the current Geometry Scale selected in the Dynamo Workspace
+    /// </summary>
+    public class GeometryScalingViewModel : NotificationObject
+    {
+        private DynamoViewModel dynViewModel;
+        private GeometryScaleSize scaleSize;
+        private double scaleValue = 0;
+        internal double ScaleValue
+        {
+            get
+            {
+                return scaleValue;
+            }
+            set
+            {
+                scaleValue = value;
+                UpdateGeometryScale(scaleValue);
+            }
+        }
+
+        internal GeometryScalingViewModel(DynamoViewModel dynViewModel)
+        {
+            this.dynViewModel = dynViewModel;
+        }
+
+        internal Tuple<string, string, string> ScaleRange
```

Comment:
could use a named tuple that is much easier to understand.

Reply:
Modified to use named tuples in the next PR: https://github.com/DynamoDS/Dynamo/pull/13734